src/Windowing/Silk.NET.Windowing.Common/Structs/WindowOptions.cs

```diff
@@ -120,26 +121,52 @@ public WindowOptions
             VSync = isVSync;
         }
 
+        static WindowOptions()
+        {
+            try
+            {
+                Default = new WindowOptions
+                (
+                    true, new Point(50, 50), new Size(1280, 720), 0.0, 0.0, GraphicsAPI.Default,
+                    Assembly.GetEntryAssembly()?.GetName().Name ?? "Silk.NET Window", WindowState.Normal,
+                    WindowBorder.Resizable, true, true, VideoMode.Default
+                );
+            }
+            catch
+            {
+                
+#if DEBUG
+                Console.WriteLine("Could not initialize WindowOptions.Default");
+#endif
+                Default = default;
```

Comment:
Please don't return default.

Reply:
What else should I return? Just some other arbitrary settings?

Reply:
The default settings we'd normally return, only with a default title as getting the title is the only thing that can throw in this.